src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -899,8 +899,11 @@ private void testResponseRate() {
         feedbacksPage.clickViewResponseLink(courseId, feedbackSessionName);
         feedbacksPage.verifyResponseValue("1 / 1", courseId, feedbackSessionName);
 
+        ______TS("check warning against changing visibility options while editing question with existing response");
+
         // Change the feedback path of the question and save
         feedbackEditPage = getFeedbackEditPage();
+        assertTrue(feedbackEditPage.verifyAlertClassIsEnabledForVisibilityOptions(1));
```

Comment:
Can we also add a check that the alert class is not enabled for a question with no existing responses (after editing this question seems like a good place as the response rate becomes 0)?